common/djangoapps/student/migrations/0047_auto__add_field_linkedinaddtoprofileconfiguration_company_identifier__.py

```diff
@@ -0,0 +1,201 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding field 'LinkedInAddToProfileConfiguration.company_identifier'
+        db.add_column('student_linkedinaddtoprofileconfiguration', 'company_identifier',
+                      self.gf('django.db.models.fields.TextField')(default='', blank=True),
+                      keep_default=False)
+
+        # Adding field 'LinkedInAddToProfileConfiguration.trk_partner_name'
+        db.add_column('student_linkedinaddtoprofileconfiguration', 'trk_partner_name',
+                      self.gf('django.db.models.fields.CharField')(default='', max_length=10, blank=True),
+                      keep_default=False)
```

Comment:
Hmm, I'm really surprised to see this.  It looks like these fields are already being added by:

https://github.com/edx/edx-platform/blob/master/common/djangoapps/student/migrations/0044_linkedin_add_company_identifier.py

https://github.com/edx/edx-platform/blob/master/common/djangoapps/student/migrations/0045_add_trk_partner_to_linkedin_config.py

There were a couple of non-standard things that happened with this migrations that might be causing this.  First, I initially deleted the `trk_partner_name`, which wasn't backwards compatible, so I replaced that migration with the current version of 0044.  Also, a separate migration from the solutions team got merged as 0044, then renamed to 0046.  Would either of those changes explain why South is trying to add these fields?